js/templates/userShort.html

```diff
@@ -0,0 +1,33 @@
+<div class="contentBox clrBr clrP clrSh2 userShort">
```

Comment:
This template will be used anywhere the small box with user info appears, such as follow, following, search results, and channels.